install/nomad

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the Apache License
+# as published by the Apache Software Foundation, either version 2 of the License, or (at your option) any later
+# version.
+
+: ${EBASH_HOME:=$(dirname $0)/..}
+: ${EBASH:=${EBASH_HOME}/share}
+source "${EBASH}/ebash.sh" || { echo "Unable to source ${EBASH}/ebash.sh" ; exit 1 ; }
+
+opt_usage main <<'END'
+This script is used to install Nomad onto a system. Darwin, alpine, arch, and gentoo will use the package manager to
+install nomad as it's a part of their repos. All other supported distributions will pull the latest Nomad binary from
+upstream.
+
+NOTE: The version downloaded is pulled using Hashicorp's checkpoint API:
+    https://checkpoint-api.hashicorp.com/v1/check/nomad
+END
+
+#----------------------------------------------------------------------------------------------------------------------
+#
+# Main
+#
+#----------------------------------------------------------------------------------------------------------------------
+
+# If we already have Nomad installed there's nothing to do.
+if command_exists nomad; then
+    edebug "nomad already installed -- nothing to do"
+    exit 0
+fi
+
+# Determine what the name of the package is to install on this OS/Distro (if any).
+package=""
+if os darwin || os_distro alpine arch gentoo; then
+    package="nomad"
+fi
+
+# Need to make sure to unmask nomad for gentoo
+if os_distro gentoo; then
+    emerge --autounmask=y --autounmask-write --autounmask-continue sys-cluster/nomad
```

Comment:
You can actually do an `exit 0` after this. It is going to unmask it and install it. If you don't want to do that, you can remove `--autounmask-continue` and it only unmask it. Then it would get installed down below.

Reply:
I dig the exit 0 here, thank ;)